xbmc/playlists/PlayListRAM.h

```diff
@@ -0,0 +1,22 @@
+/*
+ *  Copyright (C) 2024 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "PlayList.h"
+
+#include <iostream>
+
+namespace KODI::PLAYLIST
+{
+class CPlayListRAM : public CPlayList
+{
+public:
+  bool LoadData(std::istream& stream) override;
+};
+} // namespace KODI::PLAYLIST
```

Comment:
Missing trailing newline. Some ancient C compilers break without a newline (not a concern for us). As a result of these legacy limitations though, many devs have IDEs that automatically insert trailing newlines, so saving this file would dirty their git history. Missing newlines show up best in GitHub diffs so best to scan for this red flag after opening a PR.

Reply:
fixed